BorrowBookUI.java

```diff
@@ -1,117 +1,98 @@
 import java.util.Scanner;
 
-
 public class BorrowBookUI {
-	
-	public static enum UI_STATE { INITIALISED, READY, RESTRICTED, SCANNING, IDENTIFIED, FINALISING, COMPLETED, CANCELLED };
+	public static enum BorrowBookUiState {
+		INITIALISED, READY, RESTRICTED, SCANNING, IDENTIFIED, FINALISING, COMPLETED, CANCELLED
+	};
 
-	private BorrowBookControl CONTROL;
-	private Scanner input;
-	private UI_STATE StaTe;
+	private BorrowBookControl control;
+	private Scanner keybordinput;

```

Comment:
@GoJoeyGo keybordinput into keyboardInput. Please

Reply:
happy to change thank you for noticing